wallet/src/actors/worker/handlers/get_addresses.rs

```diff
@@ -9,6 +9,8 @@ pub struct GetAddresses(
     pub u32,
     /// Limit
     pub u32,
+    /// External
+    pub bool,
```

Comment:
Refactor this into a struct with name fields please

```suggestion
    /// External
    pub external: bool,
```

Reply:
Yep, that's a known a easy refactor! :)

Reply:
this is a common issue with all handlers. I would change them all in the same issue, not on this one obviously